hazelcast/src/main/resources/hazelcast-config-5.2.xsd

```diff
@@ -3313,7 +3313,8 @@
 
     <xs:complexType name="native-memory">
         <xs:all>
-            <xs:element name="size" type="memory-size" minOccurs="0"/>
+            <xs:element name="size" type="capacity" minOccurs="0"/>
```

Comment:
Maybe add an `xs:annotation/xs:documentation` child element stating this is deprecated and indicating `capacity` as the element to use in the future? (similarly elsewhere)

Reply:
Good idea, 074f8ffed1dbc57d44e8a20984538bb6591851cb